package.json

```diff
@@ -9,8 +9,8 @@
   "typings": "lib/index.d.ts",
   "scripts": {
     "lint": "tslint -p tsconfig.tslint.json src/**/*.ts test/**/*.ts",
-    "jest": "jest --ci",
-    "jest-coverage": "jest --ci --coverage",
+    "jest": "jest --testURL=\"http://localhost\" --ci",
```

Comment:
Intentional?

Reply:
I had to add this because of an error I was getting with Jest:

`SecurityError: localStorage is not available for opaque origins at Window.get localStorage [as localStorage]`

See here: https://stackoverflow.com/questions/51554366/jest-securityerror-localstorage-is-not-available-for-opaque-origins

Reply:
I think you can put that in the jest config section rather than via cli arg